app.js

```diff
@@ -57,9 +57,7 @@ function gameSetup() {
 
 function addLetterListeners() {
   const letters = document.querySelectorAll('.letter-wrapper');
-  [...letters].forEach(item => item.addEventListener('click', (event) => {
-    letterCheck(item.id);
-  }));
+  [...letters].forEach(item => item.addEventListener('click', event => letterCheck(item.id)));
```

Comment:
as you don't use `event` object here, you can skip it 
`('click', () => letterCheck(item.id)`